### ISSUE TITLE
LAPA

### ISSUE THREAD

**Issue by andrearosasco** [andrearosasco/papers#25]
#### Architecture
Basically we have a VQ-VAE composed of an encoder which takes in input an image observation x_t and the observation after H steps x_t+H and generate the latent z_t and a decoder which takes z_t and x_t and has to generate x_t+H. Basically we can see z_t as a latent representation of the action. This allows to apply this to any video without action labels.

Then we pre-train a VL model to get x_t and predict z_t. Finally we can finetune the VL model to a specific robot embedding with actual actions ground-truth and use it as a policy.

#### Takeaways 
- Pre-training LAPA with Something-Something2 we get better performance than just training on the task data

#### Benchmarks 
Sim: Language-table, SIMPLER
Real: tabletop
